meta-mender-core/classes/mender-ubimg.bbclass

```diff
@@ -72,7 +72,7 @@ vol_name=rootfsb
 
 [data]
 mode=ubi
-image=${IMGDEPLOYDIR}/${IMAGE_NAME}.dataimg
+image=${IMGDEPLOYDIR}/${IMAGE_BASENAME}-${MACHINE}.dataimg
```

Comment:
Shouldn't this match what is in `IMAGE_CMD_dataimg_mender-image-ubi`?

Reply:
yes, however IMAGE_NAME has timestamp data in it, this causes an error if you make a rebuild with yocto (it will not find it, because the timestamp changed). The only solution till now is to make a complete clean rebuild, which takes time.

With ${IMAGE_BASENAME}-${MACHINE} it points to the symlink which points to the ${IMAGE_NAME}.dataimg

Reply:
:+1: